versioned_docs/version-6.x/native-stack-navigator.md

```diff
@@ -511,41 +511,41 @@ navigation.popToTop();
 ## Example
 
 ```js
-import { createStackNavigator } from '@react-navigation/native-stack';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
 
-const Stack = createStackNavigator();
+const Stack = createNativeStackNavigator();
 
 function MyStack() {
   return (
-    <Stack.Navigator
+    <NativeStack.Navigator
```

Comment:
You forgot to change here and also Screens

Reply:
done!